cmd/discovery/cmd.go

```diff
@@ -11,12 +11,15 @@ import (
 	"net/url"
 	"os"
 	"slices"
+	"time"
 
 	"github.com/viam-modules/viamrtsp/viamonvif"
 	"github.com/viam-modules/viamrtsp/viamonvif/device"
 	"go.viam.com/rdk/logging"
 )
 
+const timeoutDuration = 10 * time.Second
```

Comment:
Curious why 10? Seems pretty long. Is this to take into account WS Discovery across the network interfaces?

Reply:
I just chose 10 to bias for having a longer timeout + it felt right. Bad reason haha, but since the cmd is a solely dev tool, I figured it'd be ok to make that decision

Reply:
Let's make this an argument input to the test script. And leave it at this default value default